charts/internal/calico/templates/custom-resource-definition/crd-bgpconfigurations.yaml

```
@@ -13,3 +14,138 @@ spec:
     kind: BGPConfiguration
     plural: bgpconfigurations
     singular: bgpconfiguration
+{{else}}
```

```suggestion
{{- else }}
```

(same for other occurrences)